YoutubeDownloader.Core/Tagging/MediaTagInjector.cs

```diff
@@ -12,7 +12,7 @@ public class MediaTagInjector
 {
     private readonly MusicBrainzClient _musicBrainz = new();
 
-    private void InjectMiscMetadata(MediaFile mediaFile, IVideo video)
+    private static void InjectMiscMetadata(MediaFile mediaFile, IVideo video)
```

Comment:
```suggestion
    private void InjectMiscMetadata(MediaFile mediaFile, IVideo video)
```

Please avoid making changes unrelated to the main goal of this PR